src/Freya.Integration/Integration.fs

```diff
@@ -136,7 +132,7 @@ module OwinMidFunc =
                 midFuncCh := midFunc.Invoke(nextSignal).Invoke(s.Environment).ContinueWith((fun _ -> false), token)
                 // If haltCh completes first, nextWasRun will be false, and nothing further is necessary.
                 // Otherwise, we'll continue on and call back into the midFunc in the after pipe.
-                let! task = Async.AwaitTask <| Task.WhenAny([| signalCh.Task; !midFuncCh |])
+                let! task = Async.AwaitTask <| Task.WhenAny ([| signalCh.Task; !midFuncCh |])
```

Comment:
Didn't you mean to replace `Task.WhenAny` with your `any` extension above? Shouldn't we move `any` to [`FSharpx.Async`](https://github.com/fsprojects/FSharpx.Async) and reference that as a dependency?